graphstorm-processing/graphstorm_processing/graph_loaders/dist_heterogeneous_loader.py

```diff
@@ -325,6 +308,8 @@ def _initialize_metadata_dict(
         return metadata_dict
 
     def _finalize_graphinfo_dict(self, metadata_dict: Dict) -> Dict:
+        if self.skip_train_masks:
+            self.graph_info["task_type"] = "link_prediction"
```

Comment:
Can we add a warning here to tell people we automatically set the task type to link prediction?

Reply:
We have a warning printed in line 165, I can add that the registered task will be LP.